src/python/grpcio/commands.py

```diff
@@ -190,8 +190,8 @@ def run(self):
                                 stderr=subprocess.STDOUT)
       except subprocess.CalledProcessError as e:
         sys.stderr.write(
-            'warning: Command:\n{}\nMessage:\n{}\nOutput:\n{}'.format(
-                command, e.message, e.output))
+            'warning: Command:\n{}\nOutput:\n{}'.format(
```

Comment:
I don't understand what the changes to this file have to do with "initial Python3 support" - perhaps embellish your commit log message with a list of "throughout the codebase, changed this thing into this other thing for this reason" items?